### PR TITLE
Githubteacher is adding a file for Fuquay Varina, NC

### LINKED ISSUE #10: Add a file for githubteacher hometown
# Adding file for Fuquay Varina, NC

- [x] Create a branch
- [x] Add a file
- [x] Commit my changes
- [x] Open a pull request
- [x] Get a code review 👀 
- [ ] Get a :+1: or :shipit: 
- [ ] Merge the pull request

### PR DESCRIPTION
**Pull request by githubteacher**
This pull request resolves #10 

@gmacario what do you like to eat and do when you travel?

### CODE REVIEW

File: dlusby-Indianapolis.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+##ndianapolis, IN
```
Comment (githubteacher) [inline dlusby-Indianapolis.md]: @dlusby it looks like the I is cut off on this line!

**Comment by ashj1** [comment]
my file was binary, I fixed it by adding a .gitattributes file.

File: githubteacher-fuquay.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Fuquay Varina, NC
+
+### Great places to eat
```
Comment (dreamchazer) [inline githubteacher-fuquay.md]: McDonald's :)

File: githubteacher-fuquay.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Fuquay Varina, NC
```
Comment (goudreau) [inline githubteacher-fuquay.md]: This is incorrect. The town name is actually "Fuquay-Varina", with a hyphen.

**Comment by leefp** [comment]
https://en.wikipedia.org/wiki/Fuquay-Varina,_North_Carolina

**Pull request merged**